react/components/ProductSummary.js

```diff
@@ -17,7 +17,7 @@ import { useCssHandles } from 'vtex.css-handles'
 const PRODUCT_SUMMARY_MAX_WIDTH = 300
 const CSS_HANDLES = ['container', 'containerNormal', 'element', 'clearLink']
 
-const ProductSummaryCustom = ({ product, actionOnClick, children }) => {
```

Comment:
por que especificamente "`inViewRef`"? pq nao só `ref`?

Reply:
For some reason, it doesn't work

Reply:
i see that our webframework doesn't seem to support `forwardRef`. 

still, `inViewRef` is too specific, I think. something like `containerRef` would be more appropriate, I believe